device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-O8V48/sai.profile

```diff
@@ -1,4 +1,4 @@
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_4700_8x400g_48x200g.xml
 SAI_DEFAULT_SWITCHING_MODE_STORE_FORWARD=1
 SAI_INDEPENDENT_MODULE_MODE=1
-SAI_NOT_DROP_SIP_DIP_LINK_LOCAL=1
\ No newline at end of file
+SAI_NOT_DROP_SIP_DIP_LINK_LOCAL=1
```

Comment:
Please don't add the empty line, just add it and remove it will fix the format.

Reply:
@bingwang-ms  Can you please recheck? I didn't add an empty line. I just fixed the format. 

```
cat device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-O8V48/sai.profile
SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_4700_8x400g_48x200g.xml
SAI_DEFAULT_SWITCHING_MODE_STORE_FORWARD=1
SAI_INDEPENDENT_MODULE_MODE=1
SAI_NOT_DROP_SIP_DIP_LINK_LOCAL=1
sudharsand@s-build-sonic-01:/builds2/sudharsand/sonic/sonic-buildimage$
```

Reply:
Thanks for replying. It's a github issue. I download the file and opened it locally, looks good.
![image](https://github.com/user-attachments/assets/d08a21c6-3e7d-4d0b-9537-2c889b67d814)